src/NHibernate.Test/NHSpecificTest/EmbIdBidirectional/Fixture.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.EmbIdBidirectional
+{
+	public class Fixture : BugTestCase
```

Comment:
The fixture needs to be marked with `[TestFixture]` for the async tests to be generated by the generator.